sauce_labs_tests/src/test/resources/mooltipass/automatedTest/features/CIFails.feature

```diff
@@ -16,4 +16,39 @@ When I login Indiegogo with 'citesting@themooltipass.com'
 Then I should be logged in Indiegogo
 When I logout Indiegogo
 And I go to Indiegogo login page
-Then I should be logged in Indiegogo
\ No newline at end of file
+Then I should be logged in Indiegogo
+
+@supported
+Scenario: Testing ldlc.ch
+Given I navigate to 'http://www.ldlc.ch/'
+When I login IDLC with 'citesting@themooltipass.com'
+Then I should be logged in IDLC
+When I logout IDLC
+And I go to IDLC login page
+Then I should be logged in IDLC
+
+@supported
+Scenario: Testing Techmania
```

Comment:
from @mosaadm : "techmania and tripadvisor are still failing" >> so why should it go there? it's still not supported.